pom.xml

```diff
@@ -604,7 +604,7 @@ THE SOFTWARE.
         <configuration>
           <signature>
             <groupId>org.codehaus.mojo.signature</groupId>
-            <artifactId>java1${java.level}</artifactId>
+            <artifactId>java16</artifactId>
```

Comment:
Need a `TODO` comment explaining that it is intended to go back to `java1${java.level}` soon.

I would actually suggest leaving `java.level` alone and just temporarily overriding compiler configuration with `<target>1.7</target>`, so that we are still using `-source 6` until the decision is finalized.